app/controllers/profiles_controller.rb

```diff
@@ -1,12 +1,28 @@
 class ProfilesController < ApplicationController
+  before_action :set_profile, only: :show
+
   def show
-    @profile = current_profile
+    @profile = @profile.decorate
+  end
+
+  def new
+    @profile = current_user.build_profile
   end
```

Comment:
Why did you include the `new` and `create` actions? There is no view for `new`

Reply:
it was part of the conflicts and it's shown here because the methods got re ordered